modules/service/src/main/resources/db/migration/V0003__functions.sql

```diff
@@ -0,0 +1,3 @@
+-- https://stackoverflow.com/questions/54372666/create-an-immutable-clone-of-concat-ws/54384767#54384767
+CREATE OR REPLACE FUNCTION immutable_concat_ws(text, VARIADIC text[])
+RETURNS text AS 'text_concat_ws' LANGUAGE internal IMMUTABLE PARALLEL SAFE;
```

Comment:
I'm using a generated column below, which is defined as a pure expression based on other fields in the row. I need to use the `text_concat_ws` function but it's not declared immutable for some reason . This basically just redeclares it with the right modifier.